src/zope/component/tests/test__api.py

```diff
@@ -114,10 +113,6 @@ class Test_queryAdapterInContext(unittest.TestCase):
 
     from zope.component.testing import setUp, tearDown
 
-    def _callFUT(self, *args, **kw):
-        from zope.component import queryAdapterInContext
-        return queryAdapterInContext(*args, **kw)
-
```

Comment:
ISTM that replacing all the imports / direct calls of `queryAdapterInContext` with `self._callFUT` would be smoother.